src/components/GiftCard/GiftCard.js

```diff
@@ -1,11 +1,15 @@
 import './GiftCard.css';
+import gift1 from "../../assets/gift1.png";
 
 const GiftCard = ({title, image}) => {
+
   return (
     <section>
       <h3>{title}</h3>
-      <img src={image}></img>
-      <button>Mark Purchased</button>
+      {/* {console.log(image)} */}
+      <img src={image} alt="Gift avatar"/>
+      {/* <img src={gift1}></img> */}
+      <button>Purchased</button>
```

Comment:
I'm sure when we get the data for the gifts we can change this to display if it has been bought or not. Also will we have the user be able to click the button to change it from Not purchased to purchased?